jmix-dynattr/dynattr/src/test/groovy/dynamic_attributes/DynAttrStringUtilsTest.groovy

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright 2024 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dynamic_attributes
+
+import io.jmix.dynattr.utils.DynAttrStringUtils
+import spock.lang.Specification
+
+class DynAttrStringUtilsTest extends Specification {
```

Comment:
I don't see tests for input that consists of several words, e.g. "Some text" -> someText or "some text" -> someText